atlp-rwanda/Avengers-Phantom-BE
# Removing all hard coded links

## What does this PR do?
Updating .env file by removing hardcoded link

## Description of Task to be completed?
I changed all file that are hardcoded and put links in .env file
 
## How should this be manually tested?

step1:cd Avengers-Phantom-BE 
step2: pull updates repo if not yet done
step3: npm i or npm install to install dependencies
step4: check code manually
